addons/website/static/src/scss/website.scss

```diff
@@ -1951,6 +1951,21 @@ $ribbon-padding: 100px;
     z-index: $zindex-modal + 1; // Over the modal backdrop.
 }
 
+// Search
+.o_searchbar_form {
+    input::-webkit-search-cancel-button  {
+        -webkit-appearance: none;
+        content: escape-svg($btn-close-bg);
+        height: $input-font-size;
+        cursor: pointer;
+    }
+
+    .dropdown-menu {
+        --dropdown-link-active-bg: #{mix($component-active-bg, $light, 30%)};
+        --dropdown-link-active-color: --dropdown-color;
```

Comment:
I don't think that this is valid CSS? Is it not missing a `var`?

Reply:
Absolutely, sorry about this :facepalm: 

I didn't notice my failing code because this line is actually useless.
Not redefining the custom property will make it have the value of the `color-contrast($component-active-bg)`, which with the default palette is white.
(taking either the component active bg contrast value from  `bootstrap_overidden.scss` or `bootstrap_overidden_frontend.scss` depending if o-cc1-primary is null)